### DISCUSSION TITLE
文章目录渲染问题

### DISCUSSION THREAD

**Discussion by zy150** [YunYouJun/valaxy#375] [Q&A]
typora写的文档目录结构正常，但是渲染后会丢失一些标题以及层级标题无法正常显示，都是一级标题

**Comment by YunYouJun** [comment]
一级标题 h1 是文章的标题，段落名称一般是二级标题 h2。

丢失是指什么呢？

**Reply by zy150** [reply]
### 本地效果
![Snipaste_2024-04-03_14-27-00](https://github.com/YunYouJun/valaxy/assets/76140031/653688e9-4c5a-4dbe-b586-146800a413ef)

---

### 博客效果
![Snipaste_2024-04-03_14-28-43](https://github.com/YunYouJun/valaxy/assets/76140031/52637531-b880-4fa7-9cec-12621048d54c)

**Reply by YunYouJun** [reply]
把 h1 变成 h2 呢？文章段落的标题应该用 H2。

**Comment by zy150** [comment]
不好意思，一直没注意规范，麻烦了
